pkg/scheduler/cache/cache.go

```diff
@@ -485,9 +485,35 @@ func (sc *SchedulerCache) setDefaultVolumeBinder() {
 	}
 }
 
-// newDefaultQueue init default queue
-func newDefaultQueue(vcClient vcclient.Interface, defaultQueue string) {
-	reclaimable := true
+// newDefaultAndRootQueue init default queue and root queue
+func newDefaultAndRootQueue(vcClient vcclient.Interface, defaultQueue string) {
```

Comment:
The main consideration here is to ensure the controller and webhook are compatible with both hierarchical queue enabled and disabled configurations. For users who do not use hierarchical queues, they should not be able to create or use the default and root queues. Do you have any suggestions regarding this?

Reply:
I think this might be better:
```
Disable hierarchy: default
Enable hierarchy: root + default
```
@hwdef Please check out this comment, what do you think?

Reply:
Because it involves the update process, after discussion, I think it is a balanced choice to create the root queue to occupy a position. Even if the hierarchical queue is not enabled,